tendrl/commons/objects/node_context/__init__.py

```diff
@@ -93,6 +93,11 @@ def render(self):
     def save(self, update=True, ttl=None):
         super(NodeContext, self).save(update)
         status = self.value + "/status"
+        if self.status == "UP" and ttl == None:
+            # Set ttl always when node status in up
```

Comment:
The comment says that ttl is set always if status is UP but condition above says status UP and ttl None

Reply:
we are already passing TTL while calling save method, but we can't always remember to pass TTL value if TTL value is not passed and status of the node is up then it will assign default TTL value.

we can pass any TTL value also. if TTL not there it will assign a default value.

Reply:
Ack